ls1intum/Artemis
# `Programming exercises`: Fix issues with hints

Exercise hints have not been usable recently. This PR fixes a couple of issues so they can be used again

### Checklist
#### General




#### Server
#### Client
#### Changes affecting Programming Exercises


### Steps for Testing

Prerequisites:
- 1 Editor
- 1 Student
- 1 Programming Exercise

1. Create a programming exercise with categories
2. Navigate into programming exercise details -> Manage hints
3. Create some hints and link them in the problem statement into tasks
4. Start the exercise as student and check if the hints are displayed properly next to the tasks

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
no changes

### Screenshots
no changes
